Texture/index.ts

```diff
@@ -1,16 +1,19 @@
-import {load_texture, loop_start} from "./core.js";
+import {create_texture_from, fetch_image} from "../common/texture.js";
+import {loop_start} from "./core.js";
 import {Game} from "./game.js";
 import {scene_stage} from "./scenes/sce_stage.js";
 
-let texture = new Image();
-texture.src = "/textures/kulka.png";
-texture.onload = () => {
-    let game = new Game();
+let game = new Game();
 
-    load_texture(game, texture);
+// @ts-ignore
+window.game = game;
+
+Promise.all([load_texture(game, "checker1.png"), load_texture(game, "kulka.png")]).then(() => {
     scene_stage(game);
     loop_start(game);
+});
 
-    // @ts-ignore
-    window.game = game;
-};
+async function load_texture(game: Game, name: string) {
```

Comment:
Should I move this to `core.ts`?

Reply:
is it `core`? Maybe we should consider adding a new namespace, like `utils`?

Reply:
Maybe once we have more utils :) It can't go to `common/` because it references `Game`, and it might also be desirable to customize the path to the `textures` dir. (And even if it's possible to modify stuff in `common/` in projects generated from goodluck, I try to keep the code there as agnostic as possible.)

Reply:
Let's keep it in `index.ts` for now.